crowbar_framework/app/models/api/node.rb

```diff
@@ -16,16 +16,20 @@
 
 module Api
   class Node < Tableless
+    attr_reader :node_name
+    attr_reader :ssh
+
     def initialize(name = nil)
-      @node = NodeObject.find_node_by_name name
+      @node_name = name
+      @ssh = Crowbar::Connection::SSH.new("root", name)
```

Comment:
Does it really make sense to initialize this here? It sounds to me like there should be some class method like: `Crowbar::Connection::SSH.exec(hostname, user, script)`